src/content/changes/highlights/2021-08-06.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2021-08-06T10:00:00
+title: Highlights for the week ending August 6, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+The unique, immutable cluster identifier formerly called the cluster ID is now called the cluster name, and what was called name is now called description. The goal here is consistency between our user interfaces and the Management API. Likewise for node pools.
+
+## Apps
```

Comment:
@ubergesundheit , @piontec , can you check the apps section to make sure you are happy with it? 🙏